lua/toggletasks/utils.lua

```diff
@@ -98,6 +98,14 @@ function M.get_work_dirs(win)
     }
 end
 
+function M.get_file_line(win)
```

Comment:
I would change the name to `get_file_cursor`/`get_win_cursor`/`get_cursor`.

Reply:
Already changed